kymatio/scattering2d/tests/test_torch_scattering2d.py

```diff
@@ -227,43 +392,58 @@ def test_Scattering2D(self, backend, device):
         M = x.shape[2]
         N = x.shape[3]
 
-        # Then, let's check when using pure pytorch code
-        scattering = Scattering2D(J, shape=(M, N), pre_pad=pre_pad, backend=backend, frontend='torch')
+        scattering = Scattering2D(J, shape=(M, N), pre_pad=pre_pad,
+                                  backend=backend, frontend='torch')
```

Comment:
remove the newline?

Reply:
Wanted to keep lines below 80 characters.

Reply:
OK.